### PR TITLE
Alertmanager: fix channel emit for alerts

### PR DESCRIPTION
**Pull request by bassel-z**
* rm Tier tag

### CODE REVIEW

File: common/prometheus-alertmanager-base/templates/alerts/_alertmanager.alerts.tpl [hunk 1]
```diff
@@ -8,9 +8,9 @@ groups:
       context: availability
       service: alertmanager
       severity: critical
-      tier: {{ include "alerts.tier" . }}
       playbook: 'docs/support/playbook/prometheus/failed_config_reload.html'
       meta: 'Alertmanager {{`{{ $labels.alertmanager }}`}} failed to load it`s configuration.'
+      support_group: {{ required ".Values.alerts.support_group missing"
```
Comment (geisslet) [inline common/prometheus-alertmanager-base/templates/alerts/_alertmanager.alerts.tpl:13]: need a fix

**Comment by bassel-z** [comment]
As discussed with @geisslet support group will be fixed to "observability"

File: common/prometheus-alertmanager-base/templates/alerts/_alertmanager.alerts.tpl (hunk 1 above)
Comment (bassel-z) [reply common/prometheus-alertmanager-base/templates/alerts/_alertmanager.alerts.tpl]: outdated

**Review by geisslet — APPROVED** [review]
lgtm for the meantime
support group should be generic in the future since this base helm is used in 2 different child setups

File: common/prometheus-alertmanager-base/templates/_helpers.tpl [hunk 2]
```diff
@@ -40,3 +40,11 @@ alertmanager-{{- (include "alertmanager.name" .) -}}
 {{- $root := index . 1 -}}
 {{- $host -}}.{{- required ".Values.global.region missing" $root.Values.global.region -}}.{{- required ".Values.global.domain missing" $root.Values.global.domain -}}
 {{- end -}}
+
+{{- define "alerts.support_group" -}}
+{{- if and .Values.global .Values.global.support_group }}
+{{- .Values.global.support_group -}}
+{{- else -}}
+{{- required ".Values.alerts.support_group missing" .Values.alerts.support_group -}}
```
Comment (viennaa) [inline common/prometheus-alertmanager-base/templates/_helpers.tpl:48]: you could also indicate either one of it is required:
```suggestion
{{- required "Either .Values.alerts.support_group or  Values.global.support_group must be set" .Values.alerts.support_group -}}
```

File: common/prometheus-alertmanager-base/templates/_helpers.tpl [hunk 3]
```diff
@@ -40,3 +40,11 @@ alertmanager-{{- (include "alertmanager.name" .) -}}
 {{- $root := index . 1 -}}
 {{- $host -}}.{{- required ".Values.global.region missing" $root.Values.global.region -}}.{{- required ".Values.global.domain missing" $root.Values.global.domain -}}
 {{- end -}}
+
+{{- define "alerts.support_group" -}}
+{{- if and .Values.global .Values.global.support_group }}
```
Comment (viennaa) [inline common/prometheus-alertmanager-base/templates/_helpers.tpl:45]:
```suggestion
{{- if Values.global.support_group | default false }}
```

**Review by viennaa — CHANGES_REQUESTED** [review]
Thanks for making this configurable. Some small suggestions to the template from my side.
Please squash the commits into one, there are 8 commits bringing in this change.

File: common/prometheus-alertmanager-base/templates/_helpers.tpl (hunk 2 above)
Comment (bassel-z) [reply common/prometheus-alertmanager-base/templates/_helpers.tpl]: Good idea. Done

File: common/prometheus-alertmanager-base/templates/_helpers.tpl (hunk 3 above)
Comment (bassel-z) [reply common/prometheus-alertmanager-base/templates/_helpers.tpl]: done

**Review by viennaa — APPROVED** [review]
Good to go from my side. Thanks. Please make sure you squash the commits with a clear commit message.

**Pull request merged**